app/views/pipelines/show.html.erb

```diff
@@ -87,15 +91,17 @@
       <% end %>
     <% end %>
```

Comment:
Probably not to fix in this story but this view is massive! Might be good to look for an opportunity to extract some partials